src/components/autocomplete/js/autocompleteController.js

```diff
@@ -203,7 +207,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       $mdUtil.enableScrolling();
     }
 
-    angular.element($window).off('resize', positionDropdown);
+    angular.element($window).off('resize', onWindowResize);
```

Comment:
Okay, so I now made the changes, because I did not see any regressions with the smoothness. 
> Please review again